### DISCUSSION TITLE
switch or light

### DISCUSSION THREAD

**Discussion by xaxexa** [arendst/Tasmota#13534] [Ideas]
we have a "setoption 30" option to select a switch or light, but what about the mask? for example I have a triple switch, 2 lights and 1 switch, I write SetOption30 101 and tasmota understands that 1 is a light, 2 is a switch, 3 is a light, a switch.light from a home assistant, a bad idea since you can not choose the location of the lamp

**Comment by barbudor** [comment]
Where did you read that '101' is a valid value?
Doc says 0 or 1
It's about how all relays on you devices are announced. Not a mask.
And it applies to relays, not switches.

At some points, complex situations need manual configuration of HA

**Reply by xaxexa** [reply]
apparently I did not explain well, I propose to add a mask, I know that it is not there
